I did those changes, but for some reason I can't test it properly, still getting:
```cmd
pnpm run docs                                               

> vite-monorepo@ docs /Users/laky/Projects/vite
> vitepress dev docs

node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/laky/Projects/vite/node_modules/.pnpm/vitepress@1.0.0-draft.8/node_modules/vite/dist/node/index.js' imported from /Users/laky/Projects/vite/node_modules/.pnpm/vitepress@1.0.0-draft.8/node_modules/vitepress/dist/node/cli.js
Did you mean to import vite/packages/vite/index.cjs?
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```
I have tried reinstall node_modules, clear everything up, git clone project again and nothing helped, I can investigate more later.

--- a/docs/guide/cli.md
+++ b/docs/guide/cli.md
@@ -115,9 +115,9 @@ vite preview [root]
 | `--strictPort`           | Exit if specified port is already in use (`boolean`) |
 | `--https`                | Use TLS + HTTP/2 (`boolean`)                         |
 | `--open [path]`          | Open browser on startup (`boolean \| string`)        |
-| `--outDir <dir>`         | Output directory (default: `dist`)(`string`)           |
+| `--outDir <dir>`         | Output directory (default: `dist`)(`string`)         |
 | `-c, --config <file>`    | Use specified config file (`string`)                 |
-| `--base <path>`          | Public base path (default: `/`) (`string`)             |
+| `--base <path>`          | Public base path (default: `/`) (`string`)           |
 | `-l, --logLevel <level>` | Info \| warn \| error \| silent (`string`)           |
 | `--clearScreen`          | Allow/disable clear screen when logging (`boolean`)  |
 | `-d, --debug [feat]`     | Show debug logs (`string \| boolean`)                |

--- a/docs/guide/cli.md
+++ b/docs/guide/cli.md
@@ -49,7 +49,7 @@ vite build [root]
 
 | Options                        |                                                                                                                     |
 | ------------------------------ | ------------------------------------------------------------------------------------------------------------------- |
-| `--target <target>`            | Transpile target (default: `'modules'`) (`string`)                                                                  |
+| `--target <target>`            | Transpile target (default: `"modules"`) (`string`)                                                                  |
 | `--outDir <dir>`               | Output directory (default: `dist`) (`string`)                                                                       |
 | `--assetsDir <dir>`            | Directory under outDir to place assets in (default: `"assets"`) (`string`)                                          |
 | `--assetsInlineLimit <number>` | Static asset base64 inline threshold in bytes (default: `4096`) (`number`)                                          |
@@ -115,9 +115,9 @@ vite preview [root]
 | `--strictPort`           | Exit if specified port is already in use (`boolean`) |
 | `--https`                | Use TLS + HTTP/2 (`boolean`)                         |
 | `--open [path]`          | Open browser on startup (`boolean \| string`)        |
-| `--outDir <dir>`         | Output directory (default: dist)(`string`)           |
+| `--outDir <dir>`         | Output directory (default: `dist`)(`string`)           |
 | `-c, --config <file>`    | Use specified config file (`string`)                 |
-| `--base <path>`          | Public base path (default: /) (`string`)             |
+| `--base <path>`          | Public base path (default: `/`) (`string`)             |
 | `-l, --logLevel <level>` | Info \| warn \| error \| silent (`string`)           |
 | `--clearScreen`          | Allow/disable clear screen when logging (`boolean`)  |
 | `-d, --debug [feat]`     | Show debug logs (`string \| boolean`)                |

